internal/schemautil/custom_diff.go

```diff
@@ -28,6 +25,13 @@ func TagsShouldNotBeEmpty(_ context.Context, _, new, _ interface{}) bool {
 	return len(new.(*schema.Set).List()) != 0
 }
 
+func CustomizeDiffServiceIntegrationAfterCreation(_ context.Context, d *schema.ResourceDiff, _ interface{}) error {
```

Comment:
What `CustomizeDiffServiceIntegrationAfterCreation` has to do with this PR?

Reply:
I just rearranged them cause we have `CustomizeDiffFunc`s mixed with a bunch of `ValueChangeConditionFunc`s, so it makes sense to group them